docs/pages/router/navigating-pages.mdx

```diff
@@ -4,6 +4,7 @@ description: Create links to move between pages.
 ---
 
 import { FileTree } from '~/ui/components/FileTree';
+import { Cell, HeaderCell, Row, Table, TableHead } from '~/ui/components/Table';
```

Comment:
I don't think we need to use these components to create a table in this markdown document. Let's remove it.